math-latex.mim

```diff
@@ -349,6 +349,62 @@ you don't have to turn off this input method to input normal ASCII characters.")
     ("<>"       "⇔")
     ("barl"     "↦")
     ("mapsto"   "↦")
+;; Superscripts
```

Comment:
Consider the s1 ... sn and v1 ... v9 codes. Given that, does this addition make sense? `[^1]` etc.. is oft used in markdown files to indicate footnotes - so these additions would inconvenience users.

Reply:
This comment made me notice the s0-s9 and v0-v9 sequences. I had not noticed them earlier, embarrassing as that is. I must have missed them while scanning the MIM file since I had already noted that the "s" key is used for script letters 𝒜-𝒵, so I assumed that the digits would be likewise treated. I think I ought to remove most of them.

However, there is a problem with keeping these sequences as-is, and there is a change still required: namely, "sn" maps to the scripts letter "𝓃", and not to the superscripted "ⁿ"; and further, that the subscript characters for letters i, a, o, etc. do not exist while the subscripts for numbers alone exist.

The subscripts are easier to handle - I can simply _append_ the additional subscript characters at the end of the subscript section On the other hand, ⁿ is harder to incorporate, since it currently clashes with 𝓃. I would suggest moving 𝓃 to "Sn" if that is all right?

Reply:
I agree with your proposed solution.